src/gui/folder.cpp

```diff
@@ -1332,22 +1338,18 @@ bool Folder::virtualFilesEnabled() const
     return _definition.virtualFilesMode != Vfs::Off && !isVfsOnOffSwitchPending();
 }
 
-void Folder::slotAboutToRemoveAllFiles(SyncFileItem::Direction dir, const std::function<void(bool)> &abort)
+void Folder::slotAboutToRemoveAllFiles(SyncFileItem::Direction direction)
 {
-    ConfigFile cfgFile;
-    if (!cfgFile.promptDeleteFiles()) {
-        abort(false);
-        return;
-    }
-    const QString msg = dir == SyncFileItem::Down ? tr("All files in the sync folder '%1' folder were deleted on the server.\n"
-                                                 "These deletes will be synchronized to your local sync folder, making such files "
-                                                 "unavailable unless you have a right to restore. \n"
-                                                 "If you decide to keep the files, they will be re-synced with the server if you have rights to do so.\n"
-                                                 "If you decide to delete the files, they will be unavailable to you, unless you are the owner.")
-                                            : tr("All the files in your local sync folder '%1' were deleted. These deletes will be "
-                                                 "synchronized with your server, making such files unavailable unless restored.\n"
-                                                 "Are you sure you want to sync those actions with the server?\n"
-                                                 "If this was an accident and you decide to keep your files, they will be re-synced from the server.");
+    const QString msg = direction == SyncFileItem::Down
+        ? tr("All files in the sync folder '%1' folder were deleted on the server.\n"
+             "These deletes will be synchronized to your local sync folder, making such files "
+             "unavailable unless you have a right to restore. \n"
+             "If you decide to keep the files, they will be re-synced with the server if you have rights to do so.\n"
+             "If you decide to delete the files, they will be unavailable to you, unless you are the owner.")
+        : tr("All the files in your local sync folder '%1' were deleted. These deletes will be "
+             "synchronized with your server, making such files unavailable unless restored.\n"
+             "Are you sure you want to sync those actions with the server?\n"
+             "If this was an accident and you decide to keep your files, they will be re-synced from the server.");
```

Comment:
:see_no_evil:

Reply:
There is an old issue that goes on and on about what to write here. Or do you men the use of the ? operator

Reply:
Yes, ? operator and = and == in one line. I dislike, because that takes longer to brain-parse than 
```
QString msg;
if (direction == SyncFileItem::Down) {
  msg = tr("...foo...");
} else {
  msg = tr("...bar...");
}
```
Yes, that means that `msg` is not `const`, but I think that is acceptable ;-)

Reply:
We usually solve this by making the code a lambda that returns the value and is called immediately. The result can be `const` again. Any compiler should optimize that out quickly, i.e., in a release build, there should be no call overhead.

Reply:
Yes, does not make the code more easily to read for "normal people", but is a clever solution of course.